src/app/submission/sections/sherpa-policies/content-accordion/content-accordion.component.scss

```diff
@@ -0,0 +1,33 @@
+:host ::ng-deep {
+    .card {
+        border: none;
+        margin-bottom: 20px;
+        border-bottom: 1px solid rgba(0, 0, 0, 0.125);
+
+        .card-header {
+            border: none;
+            background-color: transparent;
+            border-top: 1px solid rgba(0, 0, 0, 0.125);
+
+            button {
+                text-align: left;
+                padding: 0px;
+                width: 100%;
+                color: #000;
+                font-weight: 600;
+
+                .fas {
+                    background: #fff;
+                    color: #000;
+                    margin-right: 10px;
+                    height: 1.25em;
+                    display: flex;
+                    align-items: center;
+                    justify-content: center;
+                    font-size: 12px;
+                }
+            }
+        }
+    }
```

Comment:
I'm not a fan of these big css changes to the default bootstrap components in the base theme. Especially not hardcoded colors, and font-sizes etc, as they will not be influenced by the rest of the theme, and have to be modified separately every time. 

Why not use default bootstap cards, and position the button and icon using utility classes like `bg-light, `text-dark`, etc?